about.html

```diff
@@ -86,5 +86,8 @@ <h2> See past poke hunt editions </h2>
                 </p>
             </div>
         </footer>
+
```

Comment:
Kindly modify your `footer` on the about page so that is displayed correctly 🧐 As you can see, the margin: 0px rule is not being applied. Try using `!important` in this case so that it gets fixed!

![image](https://user-images.githubusercontent.com/15807343/169213011-f4889386-b133-44c9-8576-bf191d3603aa.png)


Nice! You made your background gray as the requirements specified 🤓